app/models/template_invocation.rb

```diff
@@ -0,0 +1,9 @@
+class TemplateInvocation < ActiveRecord::Base
+
+  belongs_to :template, :class_name => "JobTemplate", :foreign_key => "template_id"
+  belongs_to :job_invocation, :inverse_of => :template_invocations
+  has_many :template_invocation_input_values
+
+  alias_method :input_values, :template_invocation_input_values
```

Comment:
i assume this is just to make it shorter, would we ever need to use the setter as well (e.g. input_values=)?